app/controllers/spaces_controller.rb

```diff
@@ -32,7 +32,7 @@ def index
   # GET /spaces/1.json
   def show
     can_edit = current_user && @space.editable_by_user?(current_user)
-    has_proper_token = @space.shareable_link_enabled && @space.shareable_link_token == params[:shareable_link_token]
+    has_proper_token = @space.shareable_link_enabled && @space.shareable_link_token == request.headers['HTTP_SHAREABLE_LINK_TOKEN']
```

Comment:
I did a little digging and found this: http://stackoverflow.com/questions/2311883/authorization-header-in-ruby-on-rails-accessed-with-key-http-authorization-inste
which explains this to my satisfaction. So I have no issue with this now. Feel free to merge at will.